packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -57,12 +53,22 @@ export const AddNewTeamMembersForm = ({
 
   const router = useRouter();
   const utils = trpc.useContext();
+  const orgBranding = useOrgBranding();
 
   const showDialog = searchParams?.get("inviteModal") === "true";
   const [memberInviteModal, setMemberInviteModal] = useState(showDialog);
   const [inviteLinkSettingsModal, setInviteLinkSettingsModal] = useState(false);
 
   const { data: team, isLoading } = trpc.viewer.teams.get.useQuery({ teamId }, { enabled: !!teamId });
+  const { data: orgMembersNotInThisTeam } = trpc.viewer.organizations.getMembers.useQuery(
```

Comment:
Having the org at hand to be able to get org members not in this team to enable inviting existing members in the org